classes/class-email-service.php

```diff
@@ -178,22 +187,47 @@ public function action( $action, Backup $backup ) {
 					$error_message = ' - ' . $error_message;
 				}
 
-				$subject = sprintf( __( 'Backup of %s Failed', 'backupwordpress' ), $domain );
-
-				$message = sprintf( __( 'BackUpWordPress was unable to backup your site %1$s.', 'backupwordpress' ) . "\n\n" . __( 'Here are the errors that we\'ve encountered:', 'backupwordpress' ) . "\n\n" . '%2$s' . "\n\n" . __( 'If the errors above look like Martian, forward this email to %3$s and we\'ll take a look', 'backupwordpress' ) . "\n\n" . __( "Kind Regards,\nThe Apologetic BackUpWordPress Backup Emailing Robot", 'backupwordpress' ), home_url(), $error_message, 'backupwordpress@hmn.md' );
+				$subject = sprintf(
+					/* translators: Domain URL */
+					esc_html__( 'Backup of %s Failed', 'backupwordpress' ),
+					$domain
+				);
+
+				$message = sprintf(
+					/* translators: 1: Site URL */
+					esc_html__( 'BackUpWordPress was unable to backup your site %1$s.', 'backupwordpress' ) . "\n\n" .
+					esc_html__( 'Here are the errors that we\'ve encountered:', 'backupwordpress' ) . "\n\n" . '%2$s' . "\n\n" .
+					esc_html__( 'If the errors above look like Martian, forward this email to %3$s and we\'ll take a look', 'backupwordpress' ) . "\n\n" .
+					home_url(),
+					$error_message,
+					'backupwordpress@hmn.md'
```

Comment:
`home_url()` needs to be escaped. Is `$error_message` safe? This whole string should be wrapped in `esc_html()` instead of the individual `esc_html__()` usage.